examples/virtualFields/README.md

```diff
@@ -0,0 +1,81 @@
+# Virtual Fields Example for PayloadCMS
+
+This example demonstrates multiple use-cases for virtual fields.
+
+The following software is required:
+- Yarn or NPM
+- NodeJS version 10+
+- A Mongo Database - IMPORTANT: you need to either have MongoDB running locally, or have signed up for a free MongoDB Atlas server in order to run this example locally.
+
+## Running locally
+
+[- Step 1. Copy example](#step-1-copy-example)
+
+[- Step 2. Set up the environment variables](#step-2-set-up-the-environment-variables)
+
+[- Step 3. Add dependencies and seed data](#step-3-add-dependencies-and-seed-data)
+
+[- Step 4. Start development server](#step-4-start-development-server)
+
+### Step 1. Copy example
+
+Copy this example with [Degit](https://www.npmjs.com/package/degit) by running the following command at your terminal:
+
+```bash
+npx degit github:payloadcms/payload/examples/virtualFields
+```
+
+### Step 2. Set up the environment variables
+
+Copy the `.env.example` file in this directory to `.env`:
+
+```bash
+cp .env.example .env
+```
+
+Make sure to enter values for `PAYLOAD_SECRET` and `MONGO_URI`. If you have MongoDB running locally, then you can use the example connection string, but if you are using Mongo Atlas or similar, you'll want to fill this value in with your own connection string.
+
+### Step 3. Add dependencies and seed data
+
+Install dependencies and seed in the initial data by running:
+```bash
+yarn
+yarn seed
+```
+
+Seeding will drop any existing database and add example data. You only need to do this once on startup, or if you want to reset your data at any point.
+
+### Step 4. Start development server
+
+Start the development server with:
+```bash
+yarn dev
+```
+
```

Comment:
- [ ] If they used the seed script, what are the credentials? Lets list them here to reduce friction.

- [ ] Lets add the BeforeLogin component used in the [demo repo](https://github.com/payloadcms/public-demo/tree/master/src/components/BeforeLogin). You can conditionally render inside the component if `process.env.PAYLOAD_PUBLIC_SEED === "true"`